tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/BasicCasesTest.java

```diff
@@ -31,13 +31,13 @@
     private BMainInstance balClient;
     private String projectPath;
 
-    @BeforeClass(enabled = false)
+    @BeforeClass(enabled = true)
```

Comment:
we can remove the `(enabled = true)`, All functions are enabled by default. We don't need to specify it explicitly.

Reply:
Removed the `enabled = true` from the test cases